### ISSUE TITLE
File was not already up-to-date

### ISSUE THREAD

**Issue by Creutzx** [Creutzx/Upper_Yaquina_Figures#3]
I think I'm still having some problems with updating new versions of the .py file I'm working on. The last time I spoke to you, it looked like it was updating OK. But the code that showed up on GitHub was from an older version of the Upper_Yaquina_Figures_for_TMDL.py file. (There was a more recent version that I had edited in PyCharm. But when I did a push from the command line, it said, "Everything up-to-date." It worked OK for me to copy/paste my newer code into the editing window on GitHub, but I also know that I'm missing some part of the proper command line procedure, probably something having to do with where the repository is located on my machine.) I went back through the process to try and troubleshoot, and I think the problem started with what I did on this page (the screenshot below)
![repository_setup](https://user-images.githubusercontent.com/32490475/34496320-ece4863a-efad-11e7-82a2-4b5efdbadc03.JPG)
.

In a nutshell, my question is: which of these options should I choose when creating a repository? For whatever reason, the Git help page skips this step!

**Comment by kbrannan** [comment]
check what branch you are in on your machine. make sure that is the one you are checking for the update on GitHub
Then do a "push" and if you get an error, send a screen shot in the response to this comment. If it works, let me know.

**Comment by kbrannan** [comment]
I think what we did on your machine today addressed this issue, so I am closing it
